evaluator/opa.go

```diff
@@ -67,15 +61,15 @@ var opaConfig = `{
 }`
 
 func NewOpaEvaluator(ctx context.Context, log *logp.Logger, cfg config.Config) (Evaluator, error) {
-	server, err := StartServer(ctx, cfg)
+	// provide the OPA configuration which specifies
+	// fetching policy bundle and logging decisions locally to the console
+	path, err := filepath.Abs("bundle.tar.gz")
```

Comment:
Looks good. If the bundle isn't present on the path, an error will register, right?

Reply:
That is correct.